data/remnant/remnant missions.txt

```diff
@@ -2285,8 +2285,12 @@ mission "Remnant: Cognizance 1"
 		government "Korath"
 		system "Nenia"
 		personality entering forbearing timid uninterested surveillance staying
-		ship "Korath Raider" "Korath Raider (Ember)"
-			cargo "void sprite" 5 20
+		fleet
+			names "korath"
+			variant
+				"Korath Raider (Ember)"
+					cargo "void sprite" 5 20
```

Comment:
This isn't a thing.

Reply:
Shouldn't that be the same as mission cargo - something that will show up on the scanner?

Reply:
No. Individual NPC ships can't be given cargo. Cargo can only be given to the entire fleet.

Reply:
When I previously had this cargo being given to the fleet, ES through a bunch of errors. Moving it from the fleet to the flagship of the fleet resolved those errors.

Reply:
No error thrown =/= error solved.
It threw errors in the fleet because the syntax was wrong.

Reply:
OK, is the syntax right now?